source-salesforce/test.flow.yaml

```diff
@@ -0,0 +1,4810 @@
+---
+captures:
+  acmeCo/source-salesforce:
+    endpoint:
+        connector:
+          image: "ghcr.io/estuary/source-salesforce:v1"
+          config: config.yaml
+    bindings:
```

Comment:
nit: can leave this empty, as `bindings: []`.

Rationale: when editing / testing locally it's trivial to restore these with `flowctl raw discover`. Since we don't have a snapshot test that verifies captured documents, these aren't necessary so probably better not to have them.